gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -364,14 +364,19 @@
   public static final HadoopConfigurationProperty<Long> GCS_GRPC_WRITE_TIMEOUT_MS =
       new HadoopConfigurationProperty<>("fs.gs.grpc.write.timeout.ms", 10 * 60 * 1000L);
 
+  public static final HadoopConfigurationProperty<Long> GCS_GRPC_UPLOAD_REQUESTS_BUFFER =
+      new HadoopConfigurationProperty<>("fs.gs.grpc.write.requests.buffered", 20L);
```

Comment:
Maybe `fs.gs.outputstream.grpc.buffered.requests` will be a more consistent name w/ HTTP properties?

Reply:
Done